topcoffea/modules/corrections.py

```diff
@@ -569,8 +591,8 @@ def GetTriggerSF(year, events, lep0, lep1):
     SF_emm=np.where(events.is_eee==True,LoadTriggerSF(year,ch='3l',flav='emm')[syst](lep0.pt,lep0.eta),1.0)
     SF_mmm=np.where(events.is_eee==True,LoadTriggerSF(year,ch='3l',flav='mmm')[syst](lep0.pt,lep0.eta),1.0)
     ls.append(SF_ee*SF_em*SF_mm*SF_eee*SF_eem*SF_emm*SF_mmm)
-  ls[1]=np.where(ls[1]==1.0,0.0,ls[1]) # syst down
-  ls[2]=np.where(ls[2]==1.0,0.0,ls[2]) # syst up
+  ls[1]=np.where(ls[1]==1.0,0.0,ls[1]) # stat unc. down
+  ls[2]=np.where(ls[2]==1.0,0.0,ls[2]) # stat unc. up
   events['trigger_sf']=ls[0] #nominal
-  events['trigger_sfDown']=ls[0]+ls[1]
-  events['trigger_sfUp']=ls[0]+ls[2]
+  events['trigger_sfDown']=ls[0]-np.sqrt(ls[1]*ls[1]+0.02*0.02) # place holder: 2% systematic unc.
+  events['trigger_sfUp']=ls[0]+-np.sqrt(ls[2]*ls[2]+0.02*0.02) # place holder: 2% systematic unc.
```

Comment:
@andrea21z,  I was just looking through the changes one more time before merging, and noticed something that I have a quick question about on this line. Is it actually supposed to be `+-` here, or just `+`?

Reply:
Good catch! It's just -

Reply:
Ok, thanks for fixing it!